packages/@react-spectrum/image/docs/Image.mdx

```diff
@@ -0,0 +1,70 @@
+<!-- Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. -->
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/image';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {Image} from '@react-spectrum/image';
+```
+
+---
+category: Content
+---
+
+# Image
+
+<p>{docs.exports.Image.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Image']} />
+
+## Examples
+
+```tsx example
+<Image src="https://i.imgur.com/Z7AzH2c.png" alt="Sky and roof" />
+```
+
+## Content
+
+The Image component is used as a child to other components providing formatting
+and semantic meaning.
+
+An Image is used to add images to components. Example of usage [Dialog](Dialog.html#examples).
+
+See [the MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img) for more information.
+
+Note: An image must either have the `decorative` prop or have the `alt` prop set.
+
+### Internationalization
+
+To internationalize an Image, a localized string should be passed to the `alt` prop.
+
+## Props
+
+<PropTable component={docs.exports.Image} links={docs.links} />
+
+## Visual Options
```

Comment:
The `loaded` and `isPlaceholder` props don't have styles in the code yet so I left them out of the docs.